### ISSUE TITLE
FR: Add font-lock faces to core buttercup macros/functions

### ISSUE THREAD

**Issue by Fuco1** [jorgenschaefer/emacs-buttercup#57]
When I now read a bigger file with lots of describes and its and before-each etc it gets quite monotone and confusing to see what is going on.  I'm a very "visual" person and I like the colors to keep things readily visible.

So far I put this in my .emacs to make things easier, I think it might be a good addition overall

``` elisp
(font-lock-add-keywords 'emacs-lisp-mode '(("(\\(describe\\|buttercup-define-matcher\\) " 1 'font-lock-keyword-face)))
```

This makes the describes stand out more so I can better see where things start, especially when you nest the specs this helps.

In dash we have a customize which you can enable or disable to get font-lock for the dash macros, we could potentially do the same here.  What do you think?

**Comment by jorgenschaefer** [comment]
Sounds good! Either a customize, or some function to call to add it … if dash sets precedence, I'm fine with using the same approach. :-)

**Comment by Fuco1** [comment]
This was actually also fixed in #78 along with #72.  So good to close.
